tableauserverclient/server/endpoint/tasks_endpoint.py

```diff
@@ -0,0 +1,30 @@
+from .endpoint import Endpoint
+from .. import TaskItem, PaginationItem
+import logging
+
+logger = logging.getLogger('tableau.endpoint.tasks')
+
+
+class Tasks(Endpoint):
+    @property
+    def baseurl(self):
+        return "{0}/sites/{1}/tasks/extract-refreshes".format(self.parent_srv.baseurl,
```

Comment:
I'm pretty sure we've talked about it, so I might just be dense -- but how do we handle this when we add subscriptions? Is `Tasks` the right endpoint or should it be more specific and wrapped by a higher level endpoint?